samples/nextjs/src/components/Layout.js

```diff
@@ -11,7 +12,7 @@ const Navigation = () => {
   return (
     <div className="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-white border-bottom">
       <h5 className="my-0 mr-md-auto font-weight-normal">
-        <a className="text-dark" href="/"><img src="/sc_logo.svg" alt="Sitecore" /></a>
+        <a className="text-dark" href="/"><Image src="/sc_logo.svg" alt="Sitecore" width={221} height={48} /></a>
```

Comment:
@ambrauer Can you, please, define constants: `LOGO_SIZE = { HEIGHT: 48, WIDTH: 221 }` instead of using `magic` numbers? :)

Reply:
Ah yes, shame on me :D